.github/workflows/lint-python.yml

```diff
@@ -1,24 +1,29 @@
-name: Lint
+name: Lint Python
 on:
   push:
     branches-ignore:
       - "dependabot/*"
+    paths:
+      - '**/*.py'
+      - '.flake8'
+      - .github/workflows/lint-python.yml
   pull_request:
+    paths:
+      - '**/*.py'
+      - '.flake8'
+      - .github/workflows/lint-python.yml
 jobs:
   Python:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.7
+    - name: Set up Python
       uses: actions/setup-python@v4
-      with:
-        python-version: 3.7
```

Comment:
@cclauss I didn't think the Python need to be pinned for the linting job, but correct me if i'm wrong